### ISSUE TITLE
Can we add a interface for reset the FrameDesc

### ISSUE THREAD

**Issue by ZENOTME** [DouglasGray/xsk-rs#25]
The user will manage all `FrameDesc` returns by `Umem`. Consider the following scene:
1. It will receive the `FrameDesc` from rx
2. Free the `FrameDesc` to a user frame manager.
3. Reuse `FrameDesc` to send a message.

In phase 2, the user should reset the data len in `FrameDesc` otherwise there is an error when it is reused.
Can we provide an interface in FrameDesc like:`fn reset(&mut self)` to clear the segment length.

**Comment by DouglasGray** [comment]
makes sense at first pass. I'm currently on holiday but will take a look when I'm back next week

**Comment by DouglasGray** [comment]
please may you elaborate a bit more? if you pass the `FrameDesc` to the `Umem` it will return e.g. a `DataMut<'a>` that you may use to write data to, and which should automatically update the length. if you want to clear and reset any of the data segment (or headroom segment) you may use the `Cursor::zero_out` method
